storage/bucket.go

```diff
@@ -868,6 +873,7 @@ func newBucket(b *raw.Bucket) (*BucketAttrs, error) {
 		SoftDeletePolicy:         toSoftDeletePolicyFromRaw(b.SoftDeletePolicy),
 		HierarchicalNamespace:    toHierarchicalNamespaceFromRaw(b.HierarchicalNamespace),
 		OwnerEntity:              ownerEntityFromRaw(b.Owner),
+		IPFilter:                 b.IpFilter,
```

Comment:
Looks like this only works for JSON, not gRPC. You need to add it to the proto converter as well.